server/internal/infrastructure/wallet/btc-embedded/wallet.go

```diff
@@ -289,11 +292,11 @@ func (s *service) GenSeed(_ context.Context) (string, error) {
 }
 
 func (s *service) Create(_ context.Context, seed, password string) error {
-	return s.create(seed, password, 0)
+	return s.create(seed, password, 512)
```

Comment:
Create is meant for creating a wallet from scratch, that's why the 0 time window. Please revert:

```suggestion
	return s.create(seed, password, 0)
```